osler/inventory/utils.py

```diff
@@ -0,0 +1,11 @@
+
+
+def get_name_and_lot_from_url_query_dict(request):
```

Comment:
lmao this is just a picky comment but this is just such a long name although descriptive